src/shell/badge-wrap.tsx

```diff
@@ -5,47 +5,33 @@
  */
 import React, { FC, forwardRef } from 'react';
 
-import { Container, Text } from '@zextras/carbonio-design-system';
+import { Container, Badge } from '@zextras/carbonio-design-system';
 import styled from 'styled-components';
 
 import { BadgeInfo } from '../../types';
 
-const MiniBadge = styled(Container)<{ badge: BadgeInfo }>`
+const MiniBadge = styled(Badge)`
 	position: absolute;
 	bottom: 25%;
 	right: 25%;
 	transform: translate(30%, 30%);
-	background: ${({ badge, theme }): string => theme.palette[badge?.color ?? 'primary'].regular};
 	min-width: 0.75rem;
 	min-height: 0.75rem;
-	line-height: 0.75rem;
-	border-radius: 0.5rem;
-	user-select: none;
-	cursor: pointer;
 	pointer-events: none;
 	z-index: 99;
+	font-size: 0.625rem;
+	padding: 0.125rem;
 `;
 
-// eslint-disable-next-line react/display-name
 const BadgeWrap: FC<{ badge: BadgeInfo }> = forwardRef<HTMLDivElement, { badge: BadgeInfo }>(
 	({ badge, children }, ref) => (
 		<Container width={'3rem'} height={'3rem'} style={{ position: 'relative' }} ref={ref}>
-			{badge.show && (
-				<MiniBadge badge={badge} height="fit" width="fit">
-					{badge.showCount ? (
```

Comment:
be aware that here you are doing a BC removing the use of badge.showCount;
I think we should avoid a BC